packages/atomic/src/global/global.pcss

```diff
@@ -14,4 +14,12 @@
   .ellipsed {
     @apply whitespace-nowrap overflow-ellipsis overflow-hidden;
   }
+
+  .with-parentheses::before {
```

Comment:
Why add those parenthesis with css pseudo class like this instead of directly in JSX ?

Reply:
Because it's easier to remove / customize